service-base/src/main/java/org/eclipse/hono/service/AbstractAdapterConfig.java

```diff
@@ -369,7 +369,7 @@ public HonoConnection deviceConnectionServiceConnection() {
      * @return The properties.
      */
     @Qualifier(DeviceConnectionConstants.DEVICE_CONNECTION_ENDPOINT)
-    @ConfigurationProperties(prefix = "hono.device-connection")
+    @ConfigurationProperties(prefix = "hono.deviceConnection")
```

Comment:
My understanding is that this needs to be kebab case with dashes.

Reply:
Yes, that was brought in by the revert of commit 62e0555346215d49f446df5a763351822b7611a5, which did break the build and which also broke validation of this PR.

The next time I rebase this PR, I will drop the "revert" comment fb456eeaad2e81dea81b4e5151d4db31db9995b0.